ddl/sanity_check.go

```diff
@@ -0,0 +1,243 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ddl
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"strings"
+
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/parser"
+	"github.com/pingcap/tidb/parser/ast"
+	"github.com/pingcap/tidb/parser/model"
+	"github.com/pingcap/tidb/sessionctx"
+	"github.com/pingcap/tidb/util/sqlexec"
+)
+
+func checkRangeCntByTableIDs(physicalTableIDs []int64, cnt int64) {
```

Comment:
if it is released in the production environment. panic info will be too simple for DBA to judge the error. 

BTW, if it is just in our test environment, we can make a build tag to skip this check.

Reply:
Could we put this file code to `ddl_test.go`？Or change the file name to test specifically. Because it's only used for testing.

Reply:
> // checkHistoryJobInTest does some sanity check to make sure something is correct after DDL complete.
// It's only check during the test environment, so it would panic directly.
// These checks may be controlled by configuration in the future.

It' could be enabled in our integration test environment, even in the production environment.
For now, it is just enabled in unit test environment, so I let it panic directly.

Reply:
When will the production environment be used?